crates/bevy_math/src/direction.rs

```diff
@@ -269,6 +281,15 @@ impl approx::UlpsEq for Dir2 {
 /// A normalized vector pointing in a direction in 3D space
 #[derive(Clone, Copy, Debug, PartialEq)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
+    feature = "bevy_reflect",
+    derive(bevy_reflect::Reflect),
```

Comment:
Nit: We can avoid the `bevy_reflect::` prefix on all of these if we change the import up top to something like:

```rust
#[cfg(feature = "bevy_reflect")]
use bevy_reflect::Reflect;
#[cfg(all(feature = "bevy_reflect", feature = "serialize"))]
use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
```

This isn't mandatory, but since we'll probably need to add a separate import for `ReflectDefault` anyways, we might as well.